lua/entities/gmod_wire_expression2/core/sound.lua

```diff
@@ -178,23 +295,49 @@ e2function void soundPitch( index, pitch )
 	local sound = getSound( self, index )
 	if not sound then return end
 	
-	sound:ChangePitch( math.Clamp( pitch, 0, 255 ), 0 )
+	sound:ChangePitch( math.Clamp( pitch, 0, 300 ) / 100, 0 )
 end
 
 e2function void soundPitch( index, pitch, fadetime )
 	local sound = getSound( self, index )
 	if not sound then return end
 	
-	sound:ChangePitch( math.Clamp( pitch, 0, 255 ), math.abs( fadetime ) )
+	sound:ChangePitch( math.Clamp( pitch, 0, 300 ) / 100, math.abs( fadetime ) )
```

Comment:
So wait, ChangePitch wants a float between 0 and 3 now?? We should be keeping the E2 interface consistent, with 0-255, if possible.